opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -57,6 +57,9 @@ namespace {
         1 / ( Metric::ReservoirVolume / Metric::Time ),
         1 / Metric::Transmissibility,
         1 / Metric::Mass,
+        1, /* gas-oil ratio */
+        1, /* oil-gas ratio */
```

Comment:
Yes, oil-gas is for a gas condensate field/reservoir condensate yield
normally called CGR

Regards

David Baxendale
david.baxendale@gmail.com

On Tue, Jul 19, 2016 at 9:30 PM, jokva <notifications@github.com> wrote:

> In opm/parser/eclipse/Units/UnitSystem.cpp
> <https://github.com/OPM/opm-parser/pull/885#discussion_r71337750>:
>
> > @@ -57,6 +57,9 @@ namespace {
> >          1 / ( Metric::ReservoirVolume / Metric::Time ),
> >          1 / Metric::Transmissibility,
> >          1 / Metric::Mass,
> > +        1, /* gas-oil ratio */
> > +        1, /* oil-gas ratio */
>
> Not necessarily, from a quick glance I can only find gas-oil ratio,
> gas-liquid ratio (uses same unit as gas-oil) and liquid-liquid ratio (water
> cut). I did choose to include it for some completeness, since oil-gas has a
> dedicated entry in the manual.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OPM/opm-parser/pull/885/files/afbe2be7d7f2733224820d83c4aaabe742832e87#r71337750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARIID4wXOqh7G3jPzbrBFs2v9Z6lmG-Hks5qXNGKgaJpZM4JPsFs>
> .
>